src/less/carousel/base/carousel.less

```diff
@@ -123,6 +119,10 @@ span.carousel__container {
     display: inline-block;
 }
 
+.carousel__fade-right {
```

Comment:
Can we call this `.carousel__mask` instead. Also, we have to be wary of using direction specific terms, because of RTL. We should use "start" and "end".